easybuild/easyconfigs/l/LAMMPS/LAMMPS-22Aug2018-CrayGNU-19.10-deepmd-0.10.2.eb

```
@@ -0,0 +1,52 @@
+# Contributed by Luca Marsella and Victor Holanda Rusu (CSCS)
+easyblock = 'MakeCp'
+
+name = 'LAMMPS'
+version = '22Aug18'
+release = 'stable_22Aug2018'
+deepmd_version = '0.10.2'
+versionsuffix = '-deepmd-%s' % deepmd_version
+
+py_maj_ver = '3'
+py_min_ver = '6'
+py_rev_ver = '5.6'
+
+pyver = '%s.%s.%s' % (py_maj_ver, py_min_ver, py_rev_ver)
+pyshortver = '%s.%s' % (py_maj_ver, py_min_ver)
+
+homepage = 'http://lammps.sandia.gov/'
+description = "LAMMPS (Large-scale Atomic/Molecular Massively Parallel Simulator) is a classical molecular dynamics simulation code designed to run efficiently on parallel computers."
+
+toolchain = {'name': 'CrayGNU', 'version': '19.10'}
+toolchainopts = { 'usempi': True, 'openmp': True }
+
+source_urls = ['https://github.com/lammps/lammps/archive']
+sources = ['%s.tar.gz' % (release)]
+
+prebuildopts = ' cd ./src && '
+prebuildopts += ' cp -r $EBROOTDEEPMD/USER-DEEPMD . && '
+prebuildopts += ' export CPATH=$EBROOTDEEPMD/include/deepmd:$CPATH && '
+prebuildopts += ' make yes-standard yes-user-cg-cmm yes-user-omp yes-user-misc yes-user-deepmd && '
+prebuildopts += ' make no-voronoi no-reax no-poems no-meam no-kim no-kokkos no-mscg no-gpu no-latte && '
+prebuildopts += ' make package-update && '
+# go to folder ./lib/reax and make package reax
+prebuildopts += ' pushd ../lib/reax && make -f Makefile.gfortran && popd && '
+# create Makefile.omp and correct Makefile.mpi
+prebuildopts += ' sed -e \'s/mpicxx/CC -fopenmp/\' ./MAKE/Makefile.mpi > ./MAKE/Makefile.omp && '
+prebuildopts += ' sed -i -e \'s/mpicxx/CC/\' ./MAKE/Makefile.mpi && '
+buildopts = [ ' mpi ', ' omp ' ]
+
+builddependencies = [
+    ('cray-fftw/3.3.8.3', EXTERNAL_MODULE),
```

Since CDT 19.10, if you are using the default cray modules you should omit the version in the dependencies:
```suggestion
    ('cray-fftw', EXTERNAL_MODULE),
```